src/app/overview/status-view/status-view.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { ExplorerModel } from '../../explorer/explorer.model';
+import { ExplorerService } from '../../explorer/explorer.service';
+import { WalletModel } from '../../wallet/wallet.model';
+import { WalletService } from '../../wallet/wallet.service';
+import { RpcSend } from '../../rpc/rpc-send.model';
+import { RpcReceive } from '../../rpc/rpc-receive.model';
+
+
+export interface SendToAddressModel {
+  amount: Number;
+  destinationAddress: String;
+  feeIncluded: Boolean;
+}
+
+@Component({
+  selector: 'app-status-view',
+  templateUrl: './status-view.component.html',
+  styleUrls: ['./status-view.component.css']
+})
+export class StatusViewComponent implements OnInit {
+
+  explorer: ExplorerModel;
+  wallet: WalletModel;
+  transaction: SendToAddressModel = {
+    amount: undefined,
+    destinationAddress: undefined,
+    feeIncluded: false,
+  };
+  rpcReceive: RpcReceive;
+  qrMainAddress: string;
+
+  constructor(
+    private explorerService: ExplorerService,
+    private walletService: WalletService
+  ) {}
+
+  ngOnInit() {
+    this.getWalletOverview();
+    this.wallet = {
+      ...this.wallet,
+      mainAddress: 'NaSdzJ64o8DQo5DMPexVrL4PYFCBZqcmsW'
```

Comment:
We can remove `this.wallet` from `ngOnInit`.
We can also remove `this.qrMainAddress`

Reply:
Do we need `walletLoading()?` Because this references `this.wallet`.

Reply:
Does the `status-view.component.html` use it? If it does we need the `walletLoading()` function. Otherwise we don't

Reply:
No, but `getWalletOverview` references `this.wallet` as well. Although `getWalletOverview` is not in the `html` for  the `status-view` component.